frontend/src/pages/data-product/components/data-product-tabs/dataset-tab/dataset-tab.tsx

```diff
@@ -57,19 +54,15 @@ export function DatasetTab({ dataProductId }: Props) {
                     formItemProps={{ initialValue: '', className: styles.marginBottomLarge }}
                     form={searchForm}
                     actionButton={
-                        <Button
```

Comment:
Do we really want to remove this button? Is the shopping cart the only way to request access?
I had in my mind that the shopping cart would be for new products primarily, but it is implemented right now to only work with existing products?
Just a question